walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,76 @@
 
 package com.google.sps;
 
-import java.util.Collection;
+import java.util.*;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    int duration = (int)request.getDuration();
```

Comment:
I initially had it as long and it had compile errors because duration everywhere else is an int, added a comment to clarify!

Reply:
Understood. However, the conversion is safe not because the "duration is in int form" but because the duration (which is in hour) cannot practically be larger than 2^32. That's the main concern of long to int conversion and the comment that is needed..
BTW, can you add a space after "(int)"?

Reply:
Done!